packages/backend/src/api/request.ts

```diff
@@ -171,13 +175,28 @@ export function buildRequest(options: CreateBackendApiOptions) {
         // @ts-expect-error
         status: res?.status,
         statusText: res?.statusText,
+        clerkTraceId: getTraceId(err, res?.headers),
       };
     }
   };
 
   return withLegacyReturn(request);
 }
 
+// Returns either clerk_trace_id if present in response json, otherwise defaults to CF-Ray header
+function getTraceId(data: unknown, headers?: Headers): string | undefined {
```

Comment:
❓ Is it worth adding some unit tests? The implementation is simple, but tests can also serve as examples!

Reply:
```suggestion
function getTraceId(data: unknown, headers?: Headers): string {
```